packages/turf-transform-translate/test.js

```diff
@@ -0,0 +1,58 @@
+const test = require('tape');
+const path = require('path');
+const fs = require('fs');
+const load = require('load-json-file');
+const write = require('write-json-file');
+const helpers = require('@turf/helpers');
+const featureCollection = helpers.featureCollection;
+const point = helpers.point;
+const translate = require('./');
+
+const directories = {
+    in: path.join(__dirname, 'test', 'in') + path.sep,
+    out: path.join(__dirname, 'test', 'out') + path.sep
+};
+
+const fixtures = fs.readdirSync(directories.in).map(filename => {
+    return {
+        filename,
+        name: path.parse(filename).name,
+        geojson: load.sync(directories.in + filename)
+    };
+});
+
+test('translate', t => {
+    for (const {filename, name, geojson}  of fixtures) {
+        let {distance, direction, units, zTranslation} = geojson.properties || {};
+
+        const translated = translate(geojson, distance, direction, units, zTranslation);
+
+        // style result
```

Comment:
Wrap these styled results as their own dedicated functions (ex: `colorize(geojson)`).

That way the tests can be more readable and the `colorize` method can be reused if need be.

```javascript
function colorize(geojson) {
    // style result
    if (geojson.geometry.type === 'Point' || geojson.geometry.type === 'MultiPoint') {
        geojson.properties['marker-color'] = '#F00';
        geojson.properties['marker-symbol'] = 'star';
    } else {
        geojson.properties['stroke'] = '#F00';
        geojson.properties['stroke-width'] = 4;
    }
    return geojson;
}
```